src/App.js

```diff
@@ -1,12 +1,26 @@
 import './App.css';
+import {
+  BrowserRouter as Router,
+  Routes,
+  Route,
+} from 'react-router-dom';
+import Header from './component/Header';
+import BookList from './component/BookList';
+import Catagory from './component/Catagory';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <h1>Welocome to Book Store</h1>
-      </header>
-    </div>
+
+    <Router>
+      <Header />
+      <div className="App">
+        <Routes>
+          <Route path="/book" element={<BookList />} />
+          <Route path="/catagory" element={<Catagory />} />
+          <Route path="/" element={<BookList />} />
```

Comment:
- [ ] Please, consider that In order to not show a blank page when visiting the site for the first time, the BookList component should be the default page:
```jsx
<Route exact path="/">
   <BookList />
</Route>
```

Reply:
Hi @elmar8287 

You are right 👍 Noted and fixed